content/docs/command-reference/exp/show.md

```diff
@@ -147,109 +119,147 @@ metric or param.
-└───────────────────────┴──────────────┴─────────┴────────────────────────┴──────────────────┘
+┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━┓
+┃ neutral:**Experiment**              ┃ neutral:**Created**      ┃ metric:**roc_auc** ┃ param:**prepare.split** ┃ param:**prepare.seed** ┃ param:**featurize.max_features** ┃ param:**featurize.ngrams** ┃ param:**train.seed** ┃ param:**train.n_est** ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━┩
+│ workspace               │ -            │  0.9608 │ 0.2           │ 20170428     │ 3000                   │ 2                │ 20170428   │ 100         │
+│ master                  │ May 29, 2021 │  0.9608 │ 0.2           │ 20170428     │ 3000                   │ 2                │ 20170428   │ 100         │
+│ ├── d384680 [exp-bc055] │ Dec 17, 2021 │ 0.92333 │ 0.2           │ 20170428     │ 500                    │ 2                │ 20170428   │ 100         │
+│ ├── d7fdde2 [exp-1b262] │ Dec 17, 2021 │ 0.94713 │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 100         │
+│ └── 6b338f8 [exp-3315b] │ Dec 17, 2021 │   0.945 │ 0.2           │ 20170428     │ 2000                   │ 2                │ 20170428   │ 100         │
+└─────────────────────────┴──────────────┴─────────┴───────────────┴──────────────┴────────────────────────┴──────────────────┴────────────┴─────────────┘
+```
+
+Combine `--only-changed` with `--drop` and/or `--keep`:
+
+```dvc
+$ dvc exp show --only-changed --drop Created --keep 'train.(?!seed)'
+```
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
+┃ neutral:**Experiment**              ┃ metric:**avg_prec** ┃ metric:**roc_auc** ┃ param:**featurize.max_features** ┃ param:**train.n_est** ┃ param:**train.min_split** ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
+│ workspace               │  0.60405 │  0.9608 │ 3000                   │ 100         │ 64              │
+│ master                  │  0.60405 │  0.9608 │ 3000                   │ 100         │ 64              │
+│ ├── d384680 [exp-bc055] │  0.51799 │ 0.92333 │ 500                    │ 100         │ 64              │
+│ ├── 6b338f8 [exp-3315b] │  0.58589 │   0.945 │ 2000                   │ 100         │ 64              │
+│ └── d7fdde2 [exp-1b262] │  0.56447 │ 0.94713 │ 1500                   │ 100         │ 64              │
+└─────────────────────────┴──────────┴─────────┴────────────────────────┴─────────────┴─────────────────┘
+```
+
+Sort experiments by the `auc` metric, in descending order:
+
+```dvc
+$ dvc exp show --only-changed --sort-by=roc_auc --sort-order desc
```

Comment:
Seems unrelated? And there's already a sorting example after this.

Reply:
In the original examples there were also 2 examples for `--sort-by`. 

Only change I did here is to replace `--sort-order asc` with `--sort-order desc` because `--sort-order asc` has the same behavior as not passing the option at all, and could give the impression that you need to pass the value in order to actually sort.